### ISSUE TITLE
Training fails with other datasets

### ISSUE THREAD

**Issue by haitran14** [openai/pixel-cnn#39]
Hi,

I would like to ask about training PixelCNN++ with other data set.

Do we just need to change hyper parameters when we train PixelCNN++ with other data set, or do we need to add more regularization, or something else?

I tested with SVHN data set, but they fails to learn.
There seems to be an overfitting problem, even though dropout is enable in training.

I tried changing the initial learning rate and learning rate pattern, but it still doesn't help.
